src/main/java/org/embulk/filter/expand_json/ExpandJsonFilterPlugin.java

```diff
@@ -38,11 +38,13 @@
         @Config("expanded_columns")
         public List<ColumnConfig> getExpandedColumns();
 
-        // Time zone of timestamp columns if the value itself doesn’t include time zone description (eg. Asia/Tokyo)
         @Config("time_zone")
         @ConfigDefault("\"UTC\"")
+        @Deprecated
```

Comment:
Make sense. I fixed it on 7041e6f0e7430da7987a74aadd3b24deabfe0149
- Removed getTimeZone method
- Throw ConfigException if 'time_zone' is used in the config.